build/debian-iptables/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+timeout: 10800s
+
+substitutions:
+  { "_ARCH": "amd64",
```

Comment:
can we build images for other architectures using qemu? or is this going to be a blocking issue?

Reply:
I think we should just build for amd64 for now to keep things simple.